docker/podman-docker.sh

```diff
@@ -1,6 +1,6 @@
 # DOCKER_HOST initialization
 
-if [ -z "$DOCKER_HOST" ]; then
+if [ -z "${DOCKER_HOST-''}" ]; then
```

Comment:
this will return `''` not an empty string
with `set -u` set 
`echo "${DOCKER_HOST-''}"` vs `echo "${DOCKER_HOST-}"`